nkms/policy/models.py

```diff
@@ -114,17 +114,18 @@ class Policy(object):
     """
     _ursula = None
 
-    def __init__(self, alice, bob=None, kfrags=(constants.UNKNOWN_KFRAG,),
-                 label=None, m=None, alices_signature=constants.NOT_SIGNED):
+    def __init__(self, alice, label, bob=None, kfrags=(constants.UNKNOWN_KFRAG,),
+                 public_key=None, m=None, alices_signature=constants.NOT_SIGNED):
 
         """
         :param kfrags:  A list of KFrags to distribute per this Policy.
         :param label: The identity of the resource to which Bob is granted access.
```

Comment:
Well, the label is not necessarily the "identity of the resource", it's more a "category" or similar. Of course, if the category is unique to the resource, then it can be considered an identity, but there can be many use cases where Alice defines category labels (e.g., "healthcare_information") that may have many associated resources.

Reply:
Oh yeah, absolutely.  That language is a holdover from `path` (and then `uri`).  

There's no question that the docstrings through `policy` need a refresher at this point.